.baseline/checkstyle/checkstyle-suppressions.xml

```diff
@@ -7,6 +7,9 @@
 <!-- IMPORTANT ECLIPSE NOTE: If you change this file, you must restart Eclipse
  for your changes to take effect in its Checkstyle integration. -->
 <suppressions>
+    <!-- Eclipse/Idea APT generated code -->
+    <suppress files="[/\\]generated_src[/\\]" checks="." />
```

Comment:
If you check out the repo, run "./gradlew eclipse" and then load the generated projects into Eclipse, the generated Immutables files in "generated_src" cause Checkstyle errors and FindBugs warnings in the workspace. Adding these lines makes those errors and warnings go away.

If there's something else that can be done for the generated Eclipse project or other settings that are incorrect on my Eclipse, happy to fix that way as well, but believe strongly that having the default Eclipse projects report errors/warnings is poor form.